webpack.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  entry: './build/app/app.js',
+  output: {
+    filename: './build/public/js/app.bundle.js'
+  },
+  module: {
+    preLoaders: [],
+    loaders: [
+      // {test: /\.jsx?$/, exclude: /node_modules/, loader: 'babel'}
```

Comment:
let me know if you agree - adding babel loader dependency will work fine for me as well.